homeassistant/components/husqvarna_automower_ble/strings.json

```diff
@@ -17,5 +17,30 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
```

Comment:
Can we reuse the strings from here? 
https://github.com/home-assistant/core/blob/6b83effc5fa797c00134c1253e281f15a9b8ca3b/homeassistant/components/husqvarna_automower/strings.json#L433

Reply:
Changed.